compiler/rustc_ast/src/ast.rs

```diff
@@ -3189,9 +3189,9 @@ mod size_asserts {
     static_assert_size!(Impl, 136);
     static_assert_size!(Item, 136);
     static_assert_size!(ItemKind, 64);
-    static_assert_size!(LitKind, 24);
+    // static_assert_size!(LitKind, 32);
```

Comment:
When I was running this, it yelled at me if I had 24 and it still yelled at me if I had 32 (same for below). Why would this be called twice, and why would they differ?

Reply:
Because `stage1` is built with the previous (beta) Rust compiler which doesn't know that you changed the layout; then when `stage2` is built with `stage1` it now knows the layout has changed.

Gatting the test behind the `bootstrap` cfg should do it.
```suggestion
    #[cfg(bootstrap)]
    static_assert_size!(LitKind, 24);
    #[cfg(not(bootstrap))]
    static_assert_size!(LitKind, 32);
```

Reply:
Makes sense - thanks!